pgmq-extension/test/expected/base.out

```diff
@@ -792,6 +730,39 @@ SELECT pgmq.archive('QueueCasedPart', 1);
  t
 (1 row)
 
+-- fails with invalid queue name
+SELECT pgmq.create('dollar$-signed');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
+SELECT pgmq.create_partitioned('dollar$-signed-part');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
+-- input validation success
+SELECT pgmq.format_table_name('cat', 'q');
+ format_table_name 
+-------------------
+ q_cat
+(1 row)
+
+SELECT pgmq.format_table_name('cat-dog', 'a');
+ format_table_name 
+-------------------
+ a_cat-dog
+(1 row)
+
+SELECT pgmq.format_table_name('cat_dog', 'q');
+ format_table_name 
+-------------------
+ q_cat_dog
+(1 row)
+
+-- input validation failure
+SELECT pgmq.format_table_name('dollar$fail', 'q');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
+SELECT pgmq.format_table_name('double--hyphen-fail', 'a');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
+SELECT pgmq.format_table_name('semicolon;fail', 'a');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
+SELECT pgmq.format_table_name($$single'quote-fail$$, 'a');
+ERROR:  queue name contains invalid characters: $, ;, --, or \'
```

Comment:
If you do want to prevent `\s` from being in a table name, add a test for it.

Reply:
allowing spaces now